driver/fillers_table.c

```diff
@@ -111,27 +111,19 @@ const struct ppm_event_entry g_ppm_events[PPM_EVENT_MAX] = {
 	[PPME_SYSCALL_UNLINK_X] = {FILLER_REF(sys_single_x)},
 	[PPME_SYSCALL_UNLINKAT_E] = {FILLER_REF(sys_autofill), 2, APT_REG, {{0}, {1} } },
 	[PPME_SYSCALL_UNLINKAT_X] = {FILLER_REF(sys_single_x)},
-#ifdef CAPTURE_64BIT_ARGS_SINGLE_REGISTER
-	[PPME_SYSCALL_PREAD_E] = {FILLER_REF(sys_autofill), 3, APT_REG, {{0}, {2}, {3} } },
-#else
 	[PPME_SYSCALL_PREAD_E] = {FILLER_REF(sys_pread64_e)},
-#endif
 	[PPME_SYSCALL_PREAD_X] = {FILLER_REF(sys_read_x)},
 #ifdef CAPTURE_64BIT_ARGS_SINGLE_REGISTER
 	[PPME_SYSCALL_PWRITE_E] = {FILLER_REF(sys_autofill), 3, APT_REG, {{0}, {2}, {3} } },
 #else
 	[PPME_SYSCALL_PWRITE_E] = {FILLER_REF(sys_pwrite64_e)},
  #endif
 	[PPME_SYSCALL_PWRITE_X] = {FILLER_REF(sys_write_x)},
-	[PPME_SYSCALL_READV_E] = {FILLER_REF(sys_single)},
+	[PPME_SYSCALL_READV_E] = {FILLER_REF(sys_readv_e)},
 	[PPME_SYSCALL_READV_X] = {FILLER_REF(sys_readv_preadv_x)},
 	[PPME_SYSCALL_WRITEV_E] = {FILLER_REF(sys_writev_e)},
 	[PPME_SYSCALL_WRITEV_X] = {FILLER_REF(sys_writev_pwritev_x)},
-#ifdef CAPTURE_64BIT_ARGS_SINGLE_REGISTER
-	[PPME_SYSCALL_PREADV_E] = {FILLER_REF(sys_autofill), 2, APT_REG, {{0}, {3} } },
-#else
-	[PPME_SYSCALL_PREADV_E] = {FILLER_REF(sys_preadv64_e)},
-#endif
+	[PPME_SYSCALL_PREADV_E] = {FILLER_REF(sys_preadv_e)},
```

Comment:
so `sys_preadv_e` now replace `sys_preadv64_e` if I understood correctly... so could we remove `sys_preadv64_e`?

Reply:
probably I would rename also `sys_pread64_e` into `sys_pread_e` if it is ok for you